### DISCUSSION TITLE
關於想借用漢化做封包的疑問

### DISCUSSION THREAD

**Discussion by BonBM** [GpointChen/FFXIVChnTextPatch-GP#449] [General]
感謝GP大大一直以來的漢化
因為我本來有在巴哈做最新版本的劇情csv漢化，但之前都是借用大大的導入工具導入給別人使用
然後也因為有一些人不懂使用該工具而無法漢化
所以想問一下能不能借用GP大的現有漢化+上我自己的新漢化內容導入，整合成一個完整的封包給人使用
出處會帶上這裡的。感恩

**Comment by GpointChen** [comment]
沒問題
或者你也可以幫忙貢獻漢化XD

**Reply by BonBM** [reply]
把我的7.0插進去之後
用saintcoinach發現是這幾個任務內檔案內指令會撞
感覺是機翻沒翻到指令？
Export of quest/048/KinGba231_04827
Export of quest/048/KinGba251_04829 
Export of quest/048/KinGba341_04834
Export of quest/048/KinGbb211_04856
感覺也就跟版上任務出問題的一致了
另外大大能留個方法讓我分享翻譯csv檔給你嗎？

**Reply by BonBM** [reply]
加了~
